src/components/DayCard.js

```diff
@@ -1,12 +1,15 @@
 import styled from 'styled-components';
 import getDisplayDate from './hooks/getDisplayDate.js';
 
-export default function DayCard({ routeNumber, data }) {
+export default function DayCard({ routeNumber, data, startDate }) {
   const date = getDisplayDate(data.date);
+  const daysDiff = (new Date(data.date) - new Date(startDate)) / 86400000;
```

Comment:
what is the 8640000 here for?

Reply:
this is the amount of milliseconds of one day 😅